### DISCUSSION TITLE
Community video to encourage people to join our guild for LHD:share

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#1160] [Ideas]
People are sometimes nervous about talking on camera, so we thought we could hold up a piece of paper with writing on it. Each person could do 1 or 2 words (depending on how many words and how many people get involved) 

We could use the text (open to suggestions)

- [x] EddieHub is an inclusive community
- [ ] Where we focus
- [x] on Collaboration 1st
- [x] and Code 2nd
- [x] EddieHub won last MLH:Build hackathon
- [ ] Join us and lets win MLH:Share

**Feel free to add more words of your choice and I can add it to the end of the video**

Deadline: I would need this by Sunday 9am GMT to have time to edit it

Example (video would have music etc)

![clip](https://user-images.githubusercontent.com/624760/112493016-af81a300-8d79-11eb-98b0-a28d5d8906cb.gif)

**Comment by starlightknown** [comment]
Since there are not many people in it for now, we can make it one sentence rather than 1-2 words. If there were more people it would have looked nice with words. I remember Nick's suggestion , we could add what we learnt and built, how did we improve ourselves after LHD build and how we'll be doing in share. We can ask some people from our discord who were with us in build to share their experiences in one two lines and how are they planning for share. What do you think?

**Reply by eddiejaoude** [reply]
Good idea, just checking I understood correctly, we will do 2 videos? One now with the text above (or similar) and then one after the hackathon where people can write what they learnt and built?

**Reply by starlightknown** [reply]
Let me rephrase that, one video with people who were already in lhd build with us describing how they made progress after joining Eddiehub and section two with new ones who could do the paper one which you suggested is what I meant, if you have any other ideas, open to it

**Reply by eddiejaoude** [reply]
oh I see, as we are short on time, I was going to keep it really simple as I know we will struggle to get people to be on video

**Reply by starlightknown** [reply]
True, sorry. Cool then let's just stick to your idea. It's unique too. 
Good news - the site is complete now, it's open for improvements tho

**Reply by eddiejaoude** [reply]
Your idea is great 👍 , we can do that later on 🤓

**Reply by starlightknown** [reply]
Now we need to arrange it properly for people

**Reply by eddiejaoude** [reply]
We could maybe doing something on our community call this Saturday?

**Reply by eddiejaoude** [reply]
Oh no, there is no video on that call, it is audio only, so won't work 🤦‍♂️

**Reply by starlightknown** [reply]
We can ask some people who ever is interested to stay and switch on their cams so we can get one shot of each right? Won't take much time

**Reply by manavmodi22** [reply]
How about, @eddiejaoude you convert this into a contest and giveaway GitHub Stickers for contributing by recording a video around the topic?

**Comment by starlightknown** [comment]
Collaboration first code second is also done, you'll be receiving the video soon

**Reply by SashoStoichkov** [reply]
I've done `on collaboration first`

**Reply by SashoStoichkov** [reply]
And I'll make some extra

**Reply by eddiejaoude** [reply]
ok thanks @kcoder63 👍

**Reply by eddiejaoude** [reply]
Thanks @SashoStoichkov yep make a few, and I can pick some out. I recommend everyone having 1 page that says `EddieHub` and also put their own words they want to include

**Comment by Nitesh-thapliyal** [comment]
We can make a cool community video something like this  [video](https://www.youtube.com/watch?v=WX98l9rBHiE) 🤩

**Reply by starlightknown** [reply]
This is soo cool 😍 we can still do this maybe in between LHD

**Reply by Nitesh-thapliyal** [reply]
yeeesssss🤩

**Reply by eddiejaoude** [reply]
wow I love this!! sorry I am behind in GitHub notifications. I will organise this now!

**Reply by eddiejaoude** [reply]
Issue raised, lets do this 💪  https://github.com/EddieHubCommunity/support/issues/1215

**Comment by eddiejaoude** [comment]
Video is live https://twitter.com/eddiejaoude/status/1376132136340701188 


https://user-images.githubusercontent.com/624760/112750384-3f189300-8fc0-11eb-9c6d-13bfd46e6ad1.mp4

**Reply by Nitesh-thapliyal** [reply]
Amazing🤩
